mission/finite_state_machine/scripts/gate_choose_side.py

```diff
@@ -0,0 +1,160 @@
+from turtle import pos
```

Comment:
forward speeds in this file should be made into rosparams, so to avoid having to dive into source code to change simple behavior

Reply:
This is the case for a few of the SMs, so I wont comment on all of them.